app/i18n/fr/conf.php

```diff
@@ -133,7 +133,7 @@
 		'number' => 'Filtre n°%d',
 		'order_asc' => 'Afficher les articles les plus anciens en premier',
 		'order_desc' => 'Afficher les articles les plus récents en premier',
-		'search' => 'Recherche de « %s »',
+		'search' => 'Recherche de « %s »',
```

Comment:
Did you change the whitespace there? In any case, does not seem related to this PR

Reply:
Not on purpose. Must have happend automatically by `vscode`. I'll take a look to see if I can undo that.

Reply:
I have no idea what happend here or how to reverse it. Looking at the raw file and checking the ["hidden" characters ](https://www.soscisurvey.de/tools/view-chars.php) both versions are exactly the same.
Current `edge`: https://raw.githubusercontent.com/FreshRSS/FreshRSS/edge/app/i18n/fr/conf.php
My change: https://raw.githubusercontent.com/FreshRSS/FreshRSS/68c132bdd005d42ae92f2ee5c4f5d4e892dc7822/app/i18n/fr/conf.php

Reply:
No, the whitespace is not the same. We were using a non-breaking space. In any case, you can just avoid committing unrelated lines, but I would be curious to know what has changed it.
![image](https://github.com/FreshRSS/FreshRSS/assets/1008324/ae4e946b-5f15-47a7-ad33-44f457a930c7)

Reply:
Fixed in https://github.com/FreshRSS/FreshRSS/pull/5458/commits/926774fdce386d156373c802d950b00593a490ff

Reply:
Thanks.